es/13_3_Potenciando_Bitcoin_con_Scripts.md

```diff
@@ -0,0 +1,317 @@
+```
+
+#### Ejecuta el script de bloqueo para un Bob victimizado
+
+Lo que esto significa es que Bob tiene 24 horas para reclamar sus fondos si Alice alguna vez intenta hacer trampa, usando el `<revokeCode>` y su firma como su script de desbloqueo:
+
+```
+Script: <SigBob> <revokeCode> OP_HASH160 <revokeHash> OP_EQUAL IF <pubKeyBob> ELSE <+24Hours> OP_CHECKSEQUENCEVERIFY OP_DROP <pubKeyAlice> ENDIF OP_CHECKSIG
+Stack: [ ]
+
+Script: OP_HASH160 <revokeHash> OP_EQUAL IF <pubKeyBob> ELSE <+24Hours> OP_CHECKSEQUENCEVERIFY OP_DROP <pubKeyAlice> ENDIF OP_CHECKSIG
+Stack: [ <SigBob> <revokeCode> ]
+
+Script: <revokeHash> OP_EQUAL IF <pubKeyBob> ELSE <+24Hours> OP_CHECKSEQUENCEVERIFY OP_DROP <pubKeyAlice> ENDIF OP_CHECKSIG
+Running: <revokeCode> OP_HASH160 
+Stack: [ <SigBob> <revokeHash> ]
+
+Script: OP_EQUAL IF <pubKeyBob> ELSE <+24Hours> OP_CHECKSEQUENCEVERIFY OP_DROP <pubKeyAlice> ENDIF OP_CHECKSIG
+Stack: [ <SigBob> <revokeHash> <revokeHash> ]
+
+Script: IF <pubKeyBob> ELSE <+24Hours> OP_CHECKSEQUENCEVERIFY OP_DROP <pubKeyAlice> ENDIF OP_CHECKSIG
+Running: <revokeHash> <revokeHash> OP_EQUAL
+Stack: [ <SigBob> True ]
+
+Script: <pubKeyBob> OP_CHECKSIG
+Running: True IF
+Stack: [ <SigBob> ]
+
+Script:  OP_CHECKSIG
+Stack: [ <SigBob> <pubKeyBob> ]
+
+Script:
+Running: <SigBob> <pubKeyBob> OP_CHECKSIG
+Stack: [ True ]
+```
+
+#### Ejecuta el script de bloqueo para una Alice virtuosa
+
+Todas las transacciones de compromiso de Alice están bloqueadas con este mismo script de bloqueo, ya sea que hayan sido revocadas o no. Eso significa que la transacción de compromiso más reciente, que es la válida actualmente, también está bloqueada. Alice nunca ha enviado una nueva transacción a Bob y, por lo tanto, nunca le envió el `revokeCode` anterior.
+
+En esta situación, podría publicar virtuosamente la transacción, cerrando el canal proto-Lightning. Ella pone la transacción en la cadena y espera 24 horas. Bob no puede hacer nada al respecto porque no tiene el código de recuperación. Luego, después de la espera, Alice reclama sus fondos. (Bob hace lo mismo con su propia transacción de compromiso final).
+
+### Bloqueo con contratos codificados con bloqueo de tiempo
+
+Las Transacciones de Compromiso Revocable fueron solo un trampolín hacia Lightning. La Lightning Network real utiliza un mecanismo más complejo llamado [contrato de bloqueo de tiempo hash] (https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), o HTLC.
+
+El objetivo principal de los HTLC es crear una red integral de participantes. Las transacciones ya no son solo entre un par de participantes que han ingresado a la red juntos, sino que ahora pueden ser entre personas previamente no asociadas. Cuando se envían fondos, se crea una serie de transacciones, cada una de ellas bloqueada con un "secretHash". Cuando se revela el "secretCode" correspondiente, se puede gastar toda la cadena de transacciones. Esto es lo que permite que las transacciones singulares se conviertan realmente en una red.
```

Comment:
- `HTLCs`
- **Replace** double quotes (` " `) in `"secretHash"` and `"secretCode"` **by** backticks (`` ` ``).